packages/@uppy/core/src/Uppy.test.ts

```diff
@@ -61,6 +63,30 @@ describe('src/Core', () => {
       ).toEqual(1)
     })
 
+    it('should be able to .use() without passing generics again', () => {
+      type TestOpts = {
+        foo?: string
```

Comment:
Options should be mandatory, as they are supposed to be filled up by `defaultOptions`
```suggestion
        foo: string
        bar: string
```